### PR TITLE
fix: try calling set_code_owner inside task

### PR DESCRIPTION
**Pull request by rgraber**
**TL;DR -** Replaces set_code_owner_attribute decorator with a manual call to set_code_owner_attribute_from_module because the decorator isn't working for some reason

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by timmc-edx** [comment]
This seems to have worked: https://github.com/edx/edx-arch-experiments/pull/482
